sdk/translation/ai-translation-text-rest/samples-dev/breakSentence.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary This sample demonstrates how to make a simple call to the Azure Text Translator service to get sentences' boundaries.
+ */
+import TextTranslationClient, {
+  ErrorResponseOutput,
+  TranslatorCredential,
+  InputTextItem,
+  FindSentenceBoundariesQueryParamProperties,
+  BreakSentenceItemOutput,
+  isUnexpected,
+} from "@azure-rest/ai-translation-text";
+
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const endpoint = process.env["ENDPOINT"] || "https://api.cognitive.microsofttranslator.com";
+const apiKey = process.env["TEXT_TRANSLATOR_API_KEY"] || "<api key>";
+const region = process.env["TEXT_TRANSLATOR_REGION"] || "<region>";
+
+export async function main() {
+  console.log("== Get Sentence Boundaries sample ==");
+
+  const translateCedential = new TranslatorCredential(apiKey, region);
+  const translationClient = TextTranslationClient(endpoint, translateCedential, undefined);
```

Comment:
why do we need to pass undefined as the last parameter?
```suggestion
  const translationClient = TextTranslationClient(endpoint, translateCedential);
```

Reply:
Reoved.